repository/a.json

```diff
@@ -1989,6 +1989,17 @@
 				}
 			]
 		},
+		{
+			"name": "Asymptote Syntax",
```

Comment:
Just "Asymptote" please, language support packages are just named after the language without things like "syntax", "highlighting" etc. because it might unlock various other features as well (like completions, snippets, etc etc).